Components/Toast/Toast.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./toaststyles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <h1>Successfully Added Toast</h1>
+    <button class="toastBtn1">Add to Cart</button>
```

Comment:
camelCase can be removed.
```suggestion
    <button class="btn btn-toast-1">Add to Cart</button>
```